kernel/packages/shared/comms/index.ts

```diff
@@ -198,6 +206,60 @@ function getParcelSceneSubscriptions(): string[] {
   return ids
 }
 
+let audioRequestPending = false
+
+export function updateVoiceRecordingStatus(recording: boolean) {
+  if (recording && voiceCommunicator) {
```

Comment:
Maybe we could use a few early exits here to avoid nesting.

Reply:
Done!